File: asp/asp/Models/DOC_IMP_PRODUCT.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace asp.Models
{
    public class DOC_IMP_PRODUCT
    {
        [Key]
        public string DOC_IMP_PRODUCT_ID { get; set; }
        public string DOC_IMP_PRODUCT_CODE { get; set; }
        public string DELIVER { get; set; }
        public string RECEIVER { get; set; }
        public string RECEIVER_NAME { get; set; }
        public decimal TOTAL { get; set; }
        public string STATUS { get; set; }
        public string CREATE_ID { get; set; }
        public DateTime? CREATE_DATE { get; set; }
        public List<DOC_IMP_PRODUCT_DT> LST_DOC_IMP_PRODUCT_DT { get; set; }
        [NotMapped]
        public string XML_DOC_IMP_PRODUCT_DT { get; set; }
        public string WAREHOUSE_ID { get; set; }
    }
}

File: webadmin/src/app/core/models/DOC_IMP_PRODUCT.ts
import * as moment from 'moment';
import { DOC_IMP_PRODUCT_DT } from './DOC_IMP_PRODUCT_DT';
export class DOC_IMP_PRODUCT {
  doC_IMP_PRODUCT_ID: string | undefined;
  doC_IMP_PRODUCT_CODE: string | undefined;
  brancH_ID: string | undefined;
  deliver: string | undefined;
  receiver: string | undefined;
  receiveR_NAME: string | undefined;
  total: number | undefined;
  status: string | undefined;
  creatE_ID: string | undefined;
  creatE_DATE: moment.Moment | undefined;
  lsT_DOC_IMP_PRODUCT_DT: DOC_IMP_PRODUCT_DT[] | undefined;
  warehousE_ID: string | undefined;
}
